Vagrantfile

```diff
@@ -21,6 +21,9 @@ vagrant_boxes = {
   "jammy-m1" => "https://app.vagrantup.com/bento/boxes/ubuntu-22.04/versions/202407.22.0/providers/parallels/arm64/vagrant.box",
   "dummy" => "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box",
 }
+vagrant_box_aliases = {
+  "jammy-m1-vmware" => "bento/ubuntu22.04",
+}
```

Comment:
I resorted to this aliasing approach I could not get a box to download from a url as per vagrant_boxes - and boy did I try!!  501 responses. And that included from some of the existing vagrant_boxes urls.

But I must have a 'jammy-m1...' vagrant_box name to satisfy the conditional provision steps.

Reply:
IDK how I got anything to work with bento/ubuntu22.04 - in fact maybe I didn't and got very confused about which localrc I had sourced when my vagrant box built.

Anyway, I think this should be bento/ubuntu-22.04-arm64 (note the hyphens)

will push a fix once confirmed

Reply:
@jjhuo do you have to edit this file to test `bento/ubuntu-22.04-arm64` with parallels or can you just export VAGRANT_BOX in your localrc?